src/FlipCounterCONAINER.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
```

Comment:
The file should be named the same as the component it exports.

Including the case.